.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: android
 jdk: oraclejdk8
-sudo: false
```

Comment:
Funny story about this, I was reading about the new container stuff and thought "that looked cool!" so I made a new fork on my branch to add that, forgot to commit it, switched to this branch to add some stuff, realized I hadn't committed on the other branch, decided this was out of scope, and took it out again... tl;dr was trying to be helpful but it didn't work